### ISSUE TITLE
内存泄漏

### ISSUE THREAD

**Issue by MyNameIsKai** [skywind3000/kcp#196]
这样跑久了会有内存泄漏 直接被系统killed 是不是ikcp_send后睡眠太少了 

void update(int i)
{
	while (1) 
        {
		isleep(10);
		IUINT32 current = iclock();
		ikcp_update(kcp1, iclock());
		ikcp_update(kcp2, iclock());
	}
}

std::thread t(update, 0);
t.detach(); 
while (1) {
	ikcp_send(kcp1, buffer, 1300);
	isleep(30);
	continue;
}

**Comment by skywind3000** [comment]
你自己要控制流量啊，能这么无限发么？发不出去只是缓存在内存理，而且你在多线程里调用同一个 kcp 对象却没有做互斥锁。

**Comment by MyNameIsKai** [comment]
有互斥锁 这里省略了
